packages/web/src/pages/upload-page/forms/EditCollectionFormForUpload.tsx

```diff
@@ -0,0 +1,56 @@
+import { useCallback } from 'react'
+
+import { CollectionValues } from '@audius/common/schemas'
+import { UploadType } from '@audius/common/store'
+import moment from 'moment'
+
+import { EditCollectionForm } from 'components/edit-collection/desktop/EditCollectionForm'
+
+import { CollectionFormState } from '../types'
+
+type EditCollectionFormProps = {
+  formState: CollectionFormState
+  onContinue: (formState: CollectionFormState) => void
+}
+
+export const EditCollectionFormForUpload = (props: EditCollectionFormProps) => {
```

Comment:
a bit wordy, but probably want "form" to be the last word regardless?

Reply:
agreed it's a bit long. Prefer to be explicit here I think as we have a lot of similarly named components. Reed and 
I liked form for upload best, but can think on other names

Reply:
haha we discussed this.. but then it sounds like we're "Editing collection" for the "upload form" which is not the case? we're actually editing the collection form in preparation for upload? 
open to discussion.. the naming in this refactor was tricky. nothing we landed on sounded perfect.